plugins/modules/zos_fetch.py

```diff
@@ -589,6 +599,7 @@ def run_module():
         fail_on_missing=dict(arg_type="bool", required=False, default=True),
         is_binary=dict(arg_type="bool", required=False, default=False),
         use_qualifier=dict(arg_type="bool", required=False, default=False),
+        tmphlq=dict(type='qualifier_or_empty', required=False, default=None),
```

Comment:
There is confusion... what I thought I saw was 'tmphlq' is a string/parameter, and tmp_hlq is a dictionary. As I look more closely, I'm seeing them intermixed. I'll try to put comment notes where I see tmphlq as a dictionary.

Reply:
Resolving because I created issue https://github.com/ansible-collections/ibm_zos_core/issues/387 - it was a good comment and point, please review the issue as well @fernandofloresg , also see the other work item that came from this PRs review https://github.com/ansible-collections/ibm_zos_core/issues/386